### ISSUE TITLE
how to play ai?

### ISSUE THREAD

**Issue by cloudjanak** [BLevyCS/Scrabble#1]
is it possiable like below in web-site 
pass the current board and it will return all playable words with starting position and direction.

http://www.lexicalwordfinder.com/?game=Words_With_Friends&layout=1&tiles=gtsfsdf&board=----------------------------------------------------------------------------------------------------------------hello------------------------------------------------------------------------------------------------------------&lexicon=wwf

**Comment by Floofys** [comment]
So would it go through every tile on the board and try to make a word on every single one? That'd be the easiest way to make sure you don't miss anything, but it would take a really long time.

**Comment by cloudjanak** [comment]
have you seen the above web-site?

**Comment by Floofys** [comment]
No. I'm not going to, either. I don't usually click links that I've never seen before from complete strangers.
